resources/windows/wix/project.wxs.erb

```diff
@@ -6,7 +6,7 @@
     Name="<%= settings[:product_name] %>"
     Language="1033"
     Codepage="1252"
-    Version="<%= @version.sub(/\.g[0-9a-z]{7}$/, '') %>"
+    Version="<%= @platform.windows_style_version(@version) %>"
```

Comment:
I didn't see this method go in, but we already have a nuget version method. I'd be more comfortable if this were wix specific, since not everything windows will have the same version requirements